model/src/main/java/com/ge/predix/acs/rest/attribute/adapter/AttributesResponse.java

```diff
@@ -33,4 +39,28 @@ public String getId() {
     public void setId(final String id) {
         this.id = id;
     }
+
+    public int checkSizeLimit(final int maxNumberOfAttributes, final int maxSizeOfAttributesInBytes) {
```

Comment:
same concern about not exposing this to ACS users. Especially because these limits only apply for attributes sourced from external sources.
We should not change anything in acs-model for this.